### ISSUE TITLE
It cannot be used on IOS and keeps prompting warnings, but everything is normal on Android.

### ISSUE THREAD

**Issue by kailiming** [Heewookji/gtm#25]
It cannot be used on IOS and keeps prompting warnings, but everything is normal on Android.
What is the reason?

Console output results：

```sh
GoogleTagManager info: Loading container: GTM-*******

GoogleTagManager error: Fetching container from network failed: An SSL error has occurred and a secure connection to the server cannot be made.

GoogleTagManager info: Attempting to load default version of container GTM-*******

GoogleTagManager debug: JavaScript Runtime exception: Attempting to execute non-function main.

line:undefined

stack:

undefined

GoogleTagManager info: Processing logged event: gtm.load with parameters: (null)

GoogleTagManager warning: Event emitted before runtime was initialized.

GoogleTagManager info: Processing logged event: _vs with parameters: {

   "_o" = auto;

   "_sc" = FlutterViewController;

   "_si" = 4718958920982460028;

}

GoogleTagManager warning: Event emitted before runtime was initialized.
```

**Comment by Heewookji** [comment]
Are you still experiencing this issue?
I created a new Flutter project for iOS testing and found no issues.
Setting ATS in info.plist may solve your problem.

**Comment by kailiming** [comment]
The problem is solved, it may be due to my network.

**Comment by kailiming** [comment]
![image](https://github.com/user-attachments/assets/f0217284-5a56-44d6-a633-0f82368a960b)

I found that each event is received twice, but the parameters are only once.

**Comment by kailiming** [comment]
![image](https://github.com/user-attachments/assets/507493f3-1727-44a6-b5fa-08422f1c9fa0)

I found that each event is received twice, but the parameters are only once.

**Comment by Heewookji** [comment]
I can't reproduce the bug. 
Please create a new issue with the reproduction code.
